Mathlib/LinearAlgebra/Multilinear/Basic.lean

```diff
@@ -688,61 +709,64 @@ variable {ι₁ ι₂ ι₃ : Type _}
 
 /-- Transfer the arguments to a map along an equivalence between argument indices.
 
-The naming is derived from `finsupp.dom_congr`, noting that here the permutation applies to the
+The naming is derived from `Finsupp.domCongr`, noting that here the permutation applies to the
 domain of the domain. -/
 @[simps apply]
-def domDomCongr (σ : ι₁ ≃ ι₂) (m : MultilinearMap R (fun i : ι₁ => M₂) M₃) :
-    MultilinearMap R (fun i : ι₂ => M₂) M₃ where
+def domDomCongr (σ : ι₁ ≃ ι₂) (m : MultilinearMap R (fun _i : ι₁ => M₂) M₃) :
+    MultilinearMap R (fun _i : ι₂ => M₂) M₃ where
   toFun v := m fun i => v (σ i)
-  map_add' _ v i a b := by
+  map_add' v i a b := by
     skip
-    letI := σ.injective.decidable_eq
+    letI := σ.injective.decidableEq
     simp_rw [Function.update_apply_equiv_apply v]
     rw [m.map_add]
-  map_smul' _ v i a b := by
+  map_smul' v i a b := by
     skip
-    letI := σ.injective.decidable_eq
+    letI := σ.injective.decidableEq
     simp_rw [Function.update_apply_equiv_apply v]
     rw [m.map_smul]
 #align multilinear_map.dom_dom_congr MultilinearMap.domDomCongr
+#align multilinear_map.dom_dom_congr_apply MultilinearMap.domDomCongr_apply
 
 theorem domDomCongr_trans (σ₁ : ι₁ ≃ ι₂) (σ₂ : ι₂ ≃ ι₃)
-    (m : MultilinearMap R (fun i : ι₁ => M₂) M₃) :
+    (m : MultilinearMap R (fun _ : ι₁ => M₂) M₃) :
     m.domDomCongr (σ₁.trans σ₂) = (m.domDomCongr σ₁).domDomCongr σ₂ :=
   rfl
 #align multilinear_map.dom_dom_congr_trans MultilinearMap.domDomCongr_trans
 
 theorem domDomCongr_mul (σ₁ : Equiv.Perm ι₁) (σ₂ : Equiv.Perm ι₁)
-    (m : MultilinearMap R (fun i : ι₁ => M₂) M₃) :
+    (m : MultilinearMap R (fun _ : ι₁ => M₂) M₃) :
     m.domDomCongr (σ₂ * σ₁) = (m.domDomCongr σ₁).domDomCongr σ₂ :=
   rfl
 #align multilinear_map.dom_dom_congr_mul MultilinearMap.domDomCongr_mul
 
-/-- `multilinear_map.dom_dom_congr` as an equivalence.
+/-- `MultilinearMap.domDomCongr` as an equivalence.
 
 This is declared separately because it does not work with dot notation. -/
 @[simps apply symm_apply]
 def domDomCongrEquiv (σ : ι₁ ≃ ι₂) :
-    MultilinearMap R (fun i : ι₁ => M₂) M₃ ≃+ MultilinearMap R (fun i : ι₂ => M₂) M₃ where
+    MultilinearMap R (fun _i : ι₁ => M₂) M₃ ≃+ MultilinearMap R (fun _i : ι₂ => M₂) M₃ where
```

Comment:
I realize that when merging I was inconsistent here; do we prefer `_i` or `_`, given that Lean complains if we use `i`.

Reply:
We tend to prefer just `_`.